phidgets_analog_outputs/package.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>phidgets_analog_outputs</name>
+  <version>2.1.2</version>
```

Comment:
This number needs to match the current release, which on the `rolling` branch would be 2.3.0.